cdhweb/pages/exodus.py

```diff
@@ -0,0 +1,122 @@
+"""Exodus utilities for pages"""
+
+import json
+import logging
+import os
+
+from django.utils.html import strip_tags
+from wagtail.images.models import Image
+
+from cdhweb.pages.models import (ContentPage, HomePage, LandingPage, LinkPage,
+                                 PageIntro)
+
+
+def is_blank(content):
+    """check for if mezzanine page content is whitespace only"""
+    # remove non-breaking space, strip tags, strip whitespace
+    return not strip_tags(content.replace('&nbsp;', '')).strip()
+
+
+def convert_slug(slug):
+    """Convert a Mezzanine slug into a Wagtail slug."""
+    # wagtail stores only the final portion of a URL with no slashes
+    # remove trailing slash, then return final portion without slashes
+    return slug.rstrip("/").split("/")[-1]
+
+
+def to_streamfield(content):
+    """Output a json object for given richtext as a "migrated" streamfield."""
+    return json.dumps([{"type": "migrated", "value": content}])
+
```

Comment:
Nice refactor — good idea to pull these utility methods out. I might have called it `exodus_utils`, but I don't think it's worth changing now.